### ISSUE TITLE
Migration to Font Awesome 5. "Icon Spacing" property lost

### ISSUE THREAD

**Issue by astbarc** [elementor/elementor#8609]
## Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

## Description

After the updating process to font awesome 5 on new elementor versions (2.6-2.6.4), the icons loose their "icon spacing" property. 

If I open the page with elementor editor and save it then the icon spacing recover its value but I have to manually go thought all the pages of my site.

## Steps to reproduce

Website with an elementor version below 2.6 and with a page that contains an icon that have set the spacing value property.
Update to the a new elementor version (2.6-2.6.4) and to font awesome 5.
The value of "icon spacing" is lost.

## Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a default WordPress theme active.
- [X] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by shilo-ey** [comment]
Thanks for reporting.

We will address this issue soon and release a patch.

Please follow this thread for updates.

Thanks!

**Comment by astbarc** [comment]
That's great.
Many thanks @shilo-ey

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v2.6.6

Feel free to update

Thanks!
